app/views/form/_select_question.html.erb

```diff
@@ -1,20 +1,23 @@
-
 <% selected = @log.public_send(question.id) || "" %>
```

Comment:
Is this file just formatting?

Reply:
I mean if it's redundant, could remove, but if there are situations where this would catch something then feel free to leave